### DISCUSSION TITLE
try to render page by javet

### DISCUSSION THREAD

**Discussion by zahrahariry** [caoccao/Javet#159] [General]
I'm trying to render a page with a javet. But I did not succeed. The site opens well with the Chrome browser, but when using javet, a regular expression used in the site's JavaScript gets an error.
Regular expression used: / [\ p {Lu}] / \
The reason for this is not clear, as both browsers and javet use v8 to run JavaScript.
thanks

**Comment by caoccao** [comment]
Are you trying to get SSR work? Usually, the V8 mode lacks of DOM and is not suitable for SSR. If you go with the Node.js mode, I suggest you make the SSR work in Node.js, then try Javet Node.js mode. That will be much easier.
